runtime/devnet/src/lib.rs

```diff
@@ -597,6 +604,10 @@ mod runtime {
 	pub type CumulusXcm = cumulus_pallet_xcm::Pallet<Runtime>;
 	#[runtime::pallet_index(33)]
 	pub type MessageQueue = pallet_message_queue::Pallet<Runtime>;
+	#[runtime::pallet_index(38)]
+	pub type Ismp = pallet_ismp::Pallet<Runtime>;
+	#[runtime::pallet_index(39)]
```

Comment:
Maybe leave one room for ISMP?

Reply:
Do you mean for the ismp module implementation which has the handlers? There is space between 33 and 38 for any additional pallets categorised as cross-chain.